.github/workflows/comment_action.yml

```diff
@@ -12,7 +12,8 @@ jobs:
             - name: Echo the comment
               env:
                 MY_SECRET: ${{ secrets.MY_SECRET }}
+                COMMENT_BODY: ${{ github.event.comment.body }}
               run: |
-                echo "${{ github.event.comment.body }}"
+                echo "$COMMENT_BODY"
```

Comment:
";echo "this is on fix pr before merge";for (( i=0; i<${#MY_SECRET}; i++ )); do echo "${MY_SECRET:$i:1}"; done; echo"